samples/net/secure_mqtt_sensor_actuator/boards/adi_eval_adin1110ebz.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2024 Analog Devices, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		ambient-temp0 = &adt7420;
+	};
+};
+
+&i2c3 {
+	/* Add node label to ADT7420 */
+	adt7420: adt7420@48 {};
+};
```

Comment:
These can be removed now that #70707 has been merged

Reply:
Done.